events/wc_cenarion_circle_events.txt

```diff
@@ -205,6 +208,25 @@ character_event = {
 		move_character = FROM
 	}
 }
+character_event = {
+	id = WCCC.23
+
+	is_triggered_only = yes
+
+	immediate = {
+	}
+	
+	option = {
+		name = EVTOPTA_WCCC.23
+		e_kaldorei = { holder = { make_tributary = { who = ROOT tributary_type = part_of_society} } }
+		ai_chance = {
+			factor = 1
+		}
+	}
+	option = {
+		name = EVTOPTB_WCCC.23
```

Comment:
Has no localisation.
```suggestion
		name = EVTOPTB_WCCC.23
		
		ai_chance = {
			factor = 0
		}
```
AI can still pick it, if there's no factor 0.

Reply:
fixed